components/server-startup-observer/src/main/java/com/wso2telco/dep/server/startup/observer/HubStartupObserver.java

```diff
@@ -194,16 +200,44 @@ private void createDefaultRolesAndAssignToSuperAdmin () {
                 if (log.isDebugEnabled()) {
                     log.debug("Creating " + MANAGE_APP_ADMIN_ROLE + " role: " + role);
                 }
-                Permission[] loginPermission = new Permission[]{new Permission("/permission/admin/login", "ui.execute")};
+                ArrayList<Permission> permissions = new ArrayList<Permission>();
+                permissions.add(new Permission("/permission/admin/login", "ui.execute"));
+                
+                addManagePermissions(permissions);
+                
                 String superTenantName = ServiceReferenceHolder.getInstance().getRealmService().getBootstrapRealmConfiguration().getAdminUserName();
                 String[] userList = new String[]{superTenantName};
-                manager.addRole(role, userList, loginPermission);
+                manager.addRole(role, userList, (permissions.toArray(new Permission[permissions.size()])));
             }
         } catch (Exception e) {
             log.error("Error in assigning 'manage-app-admin' role to super tenant", e);
         }
     }
 
+    private void addManagePermissions(ArrayList<Permission> permissions) {
```

Comment:
Better to handle common permission for all deployments separately to avoid the code duplication and define constants instead hard codes values

Reply:
Can define constants. 

Not considering common permissions etc as this has been done only for EGW. When it is being implemented for another flavour then we will have to move out the common permissions.